_studio/mfx_lib/plugin/src/mfx_hevc_enc_plugin.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2017 Intel Corporation
+// Copyright (c) 2019 Intel Corporation
```

Comment:
often we do ```2017-2019``` in such cases. But I don't think if particular format matters. @onabiull can you comment here?